### ISSUE TITLE
Trained models

### ISSUE THREAD

**Issue by luke-777** [zhaoyuzhi/Semantic-Colorization-GAN#4]
Hi,

can you please share the trained models (vgg16_bn_fc_gray_epoch150_bs256.pth,...) ?

Thank you!

**Comment by zhaoyuzhi** [comment]
Hi,

Could you please refer to https://github.com/zhaoyuzhi/PyTorch-Special-Pre-trained-Models?

Thanks

**Comment by luke-777** [comment]
Thank you,
which weights do I have to use for following 2 arguments?

```
parser.add_argument('--global_feature_network_path', type = str, \
       default = './trained_models/vgg16_bn_fc_gray_epoch150_bs256.pth', \
       help = 'the path that contains the pre-trained ResNet model')

parser.add_argument('--perceptual_path', type = str, default = './trained_models/vgg16_pretrained.pth', help = 'the path that contains the pre-trained VGG-16 model')
```

**Comment by zhaoyuzhi** [comment]
`vgg16_bn_fc_gray_epoch150_bs256.pth` is a pre-trained network embedded in SCGAN and you can refer to https://github.com/zhaoyuzhi/PyTorch-Special-Pre-trained-Models

`vgg16_pretrained.pth` is also a pre-trained network, but it is used to compute perceptual loss. It can be downloaded from torchvision
